geth/notification/notification_test.go

```diff
@@ -0,0 +1,78 @@
+package notification
+
+import (
+	"testing"
+
+	"errors"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/notification/message"
+	t "github.com/status-im/status-go/geth/testing"
```

Comment:
Confusing with `t *testing.T`. I would stay with `testing`. It's still pretty compact :)

Reply:
Little bit higher there is already import of standard 'testing' package.